README.md

```diff
@@ -28,11 +28,18 @@ git clone https://github.com/SovereignCloudStack/rookify
 cd rookify
 ```
 
-3. __TODO:__ Install script
+3. For local installation of a development setup:
+```bash
+make setup
+```
+
```

Comment:
We should add some note like:
_Note:_ This will install `pre-commit` in your local user context.

Reply:
ok added that